Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -1641,9 +1651,10 @@ bool CSSPrimitiveValue::equals(const CSSPrimitiveValue& other) const
     case CSSUnitType::CSS_STRING:
     case CSSUnitType::CustomIdent:
     case CSSUnitType::CSS_URI:
-    case CSSUnitType::CSS_ATTR:
     case CSSUnitType::CSS_FONT_FAMILY:
         return equal(m_value.string, other.m_value.string);
+    case CSSUnitType::CSS_ATTR:
+        return equal(m_value.attr->cssText(), other.m_value.attr->cssText());
```

Comment:
Comparing the cssText() seems wrong, I would define a `operator==` on CSSAttrValue (that's just the default one) and use that here.